listenbrainz/webserver/views/index.py

```diff
@@ -34,11 +37,14 @@ def index():
     except Exception as e:
         current_app.logger.error('Error while trying to get total listen count: %s', str(e))
         listen_count = None
-
-
+    try:
+        latest_listens = store_latest_listen()
```

Comment:
This function isn't imported from anywhere?

Reply:
Added the import